src/ray/raylet/main.cc

```diff
@@ -121,15 +125,17 @@ absl::flat_hash_map<std::string, std::string> parse_node_labels(
 }
 
 int main(int argc, char *argv[]) {
+  gflags::ParseCommandLineFlags(&argc, &argv, true);
   InitShutdownRAII ray_log_shutdown_raii(ray::RayLog::StartRayLog,
                                          ray::RayLog::ShutDownRayLog,
                                          argv[0],
                                          ray::RayLogLevel::INFO,
-                                         /*log_dir=*/"");
+                                         /*log_dir=*/"",
+                                         /*ray_log_filepath=*/FLAGS_ray_log_filepath);
```

Comment:
We need to do the same setting rotation bytes to LONG::MAX by default?

Reply:
Yes.